public/resources/remix/style.less

```diff
@@ -272,3 +272,21 @@
     }
   }
 }
+/* move the remix button to the left side for AFrame projects only*/
+.a-body .details-bar.touch-mode {
+  right: auto !important;
+  left: 6px !important;
+    
+  .thimble-button {
+    left: 0px !important;
+  }
+    
+  .details-bar-remix-button-wrapper {
+    margin-right: auto !important;
+    margin-left: 46px !important;
+  }
+
+  .close-details-bar {
+    left: 10px !important;
+  }
+}
```

Comment:
I made the change but still have a question: Is this a good parctice that adding a line break at the end of the file? If it is, I would be really carefully with the fix that I'm doing right now. Thank you.